community/QClass_2024/Submissions/HW3/Razeen_Ud_Din_HW3_QClass2024.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<p>That is two lines :)</p><p>Check the solutions which soon will be uploaded!</p><p> </p><p>[Genrally</p><p>U...|</p><p>U...</p><p>CX...</p><p>]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>